components/time-picker/time-picker-panel.component.spec.ts

```diff
@@ -459,6 +471,7 @@ export class NzTest12HourTimePanelDisabeledComponent {
   @ViewChild(NzTimePickerPanelComponent, { static: false }) nzTimePickerPanelComponent!: NzTimePickerPanelComponent;
   format = 'hh:mm:ss a';
   value = new Date(0, 0, 0, 1, 1, 1);
+  hideDisabledOptions = false;
```

Comment:
If `hideDisabledOptions` should be always false, why don'y you set `[nzHideDisabledOptions]="hideDisabledOptions"` to be `[nzHideDisabledOptions]="false"`?

Reply:
thanks, i have updated it.